tests/TextInjection1Test.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace nadar\quill\tests;
+
+class TextInjection1Test extends DeltaTestCase
+{
+    public $json = <<<'JSON'
+[
+    {"insert":"normal text"},
+    {"insert":"<script type='text/javascript'>alert(\"html injection\");</script>"},
+    {"insert":"bold text", "attributes":{"bold":true}},
+    {"insert":"\n"}
+]
+JSON;
+
+    public $html = <<<'EOT'
+<p>normal text</p><p>&lt;script type=&apos;text/javascript&apos;&gt;alert(&quot;html injection&quot;);&lt;/script&gt;<strong>bold text</strong></p>
```

Comment:
I didn't expect the `</p><p>` here directly after `normal text`. Is that because somewhere the parser recognizes the opening `<script>` tag?

Reply:
I assume unless you are not adding attributes there is no reason for a new insert key:

{"insert":"normal text"},
{"insert":"<script type='text/javascript'>alert(\"html injection\");</script>"},

it should be

 {"insert":"normal text<script type='text/javascript'>alert(\"html injection\");</script>"},